src/lib/OpenEXR/ImfCompression.cpp

```diff
@@ -176,6 +176,30 @@ static const CompressionDesc IdToDesc[] = {
         256,
         true,
         false),
+    CompressionDesc (
+        "ht",
+        "High-Throughtput JPEG 2000 (OpenJPH, full frame)",
+        16000,
```

Comment:
```suggestion
        16384,
```
16384 lines?

Reply:
16000 is just a placeholder to ensure that the full-frame is encoded in one chunk and thus understand the impact of full-frame vs chunk encoding on coding efficiency.